### PR TITLE
[FEATURE/BALANCE] Parrying with Chairs and Fighting Lions!

### PR DESCRIPTION
**Pull request by Spaghetti-bit**
## What Does This PR Do

Allows chairs, stools, and other bar brawling essential sitting devices to be wielded and allows them to parry melee attacks.

Wielding a chair allows you to do... ~20% more damage, parrying with a chair is god awful and will drain you of stamina... very fast unless you perfectly parry it.

You now set down chairs by Alt-clicking them.
## Why It's Good For The Game

Adds more finesse to that random assistant denizen. Makes bar brawls more interesting.
## Images of changes

image

## Testing

Loaded an offline server, angered pete, kept him at bay with a bar stool.

## Changelog
:cl:
add: Chair parrying!
add: Chair wielding!
tweak: Chairs now have to be alt-clicked to be placed down, but also do not have to be held to upright them
/:cl:

### CODE REVIEW

**Comment by Ionward** [comment]
you could always wield them before, chairs had a good knockdown chance and also could break when you swung em, but parrying is fun and a very classic visual

**Comment by Spaghetti-bit** [comment]
> you could always wield them before, chairs had a good knockdown chance and also could break when you swung em, but parrying is fun and a very classic visual

Wield as in two handed wielding.

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 1]
```diff
@@ -464,14 +464,33 @@
...
 	w_class = WEIGHT_CLASS_HUGE
-	force = 8
+	force = 6
 	throwforce = 10
 	throw_range = 3
 	hitsound = 'sound/items/trayhit1.ogg'
 	hit_reaction_chance = 50
 	materials = list(MAT_METAL = 2000)
-	var/break_chance = 5 //Likely hood of smashing the chair.
+	/// Likely hood of smashing the chair.
```
Comment (Fordoxia) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:473]:
```suggestion
	/// Likelihood of smashing the chair.
```

**Review by Fordoxia — APPROVED** [review]
Aside from this one thing, looks good.

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 2]
```diff
@@ -501,7 +522,8 @@
 	item_state = "stool_bamboo"
 	origin_type = /obj/structure/chair/stool/bamboo
 
-/obj/item/chair/attack_self(mob/user)
+/obj/item/chair/AltClick(mob/user)
+	. = ..()
 	plant(user)
```
Comment (Sirryan2002) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:527]: Did you test to make sure that players can't remotely (i.g. from more than 1 tile away) plant the chair just by alt clicking it from a distance?

File: code/game/objects/structures/stool_bed_chair_nest/chairs.dm [hunk 3]
```diff
@@ -560,6 +582,10 @@
 		return
 	..()
 
+/obj/item/examine(mob/user)
+	. = ..()
+	. += "<span class='info'>You can <b>Alt-Click</b> [src] to place it down.</span>"
+
```
Comment (Sirryan2002) [inline code/game/objects/structures/stool_bed_chair_nest/chairs.dm:588]: Perhaps you should check the typepath here :)

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

**Comment by Sirryan2002** [comment]
@Spaghetti-bit

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**